community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -70,6 +69,13 @@
  */
 public abstract class GraphDatabaseSettings
 {
+    /**
+     * Data block sizes for dynamic array stores.
+     */
+    public static final int DEFAULT_BLOCK_SIZE = 128;
+    public static final int DEFAULT_LABEL_BLOCK_SIZE = 64;
+    public static final int MINIMAL_BLOCK_SIZE = 16;
+
```

Comment:
they where here before as default values of properties, now they represented as a constants that will be used to evaluate correct format specific value, so they are public as they were before